src/scenes/levelCountdown.js

```diff
@@ -0,0 +1,60 @@
+let timer = 3; //Countdown timer
+let midLine; //Horizontal line in the scene
+let midLine2; //Vertical line in the scene
+let animatedLine; //Line that will circle around during the countdown
+let angle=0; //Angle the animatedLine will use to circle around
+let finishedCountdown = false; //Boolean to stop the line from circling around when countdown stops
+
+const levelCountdown = function()
```

Comment:
The way this is structured will be fixed when the code has been refactored into modules. Look to other files upon being lost. With the implementation, looks good.